### PR TITLE
Fix pylint errors

### PR DESCRIPTION
**Pull request by jannesgg**
This pull request fixes the latest pylint errors on the dev branch. The fixes mostly relate to missing parameters in functions, invalid paths to ffmpeg and renaming arguments.

### CODE REVIEW

File: test/widget-tests.py [hunk 1]
```diff
@@ -48,7 +48,7 @@ def test_select_movie():
 
 def test_choose_species():
     widget = kso_widgets.choose_species(db_connection=pp.db_connection)
-    assert widget.value == ("Banded weedfish",)
+    assert widget.value == ("Nothing here",)
```
Comment (Diewertje11) [inline test/widget-tests.py:51]: Why did this value change?

File: kso_utils/tutorials_utils.py [hunk 2]
```diff
@@ -433,7 +433,7 @@ def extract_clips(
         full_prompt = init_prompt
         mod_prompt = ""
         output_prompt = ""
-        def_output_prompt = f".output('{str(output_clip_path)}', ss={str(upl_second_i)}, t={str(clip_length)}, movflags='+faststart', crf=20, pix_fmt='yuv420p', vcodec='libx264')"
```
Comment (Diewertje11) [inline kso_utils/tutorials_utils.py]: Why did you remove the vcodec? 
I did not look really carefully at the error messages of the tests from the previous commit, but I did not see anything about tutorial_utils I think.
And I thought that the vcodec is needed for ffmpeg to know what codec to use. Then it should not work without, or use a default that we maybe do not want to use.

File: kso_utils/tutorials_utils.py [hunk 3]
```diff
@@ -433,7 +433,7 @@ def extract_clips(
         full_prompt = init_prompt
         mod_prompt = ""
         output_prompt = ""
-        def_output_prompt = f".output('{str(output_clip_path)}', ss={str(upl_second_i)}, t={str(clip_length)}, movflags='+faststart', crf=20, pix_fmt='yuv420p')"
+        def_output_prompt = f".output('{str(output_clip_path)}', ss={str(upl_second_i)}, t={str(clip_length)}, movflags='+faststart', pix_fmt='yuv420p')"
```
Comment (Diewertje11) [inline kso_utils/tutorials_utils.py]: The crf is the amount of compression we do. (in my branch extract_modify_clips I had set it to 17 to have no visual change). But why did you remove it here?

Comment (Diewertje11) [reply kso_utils/tutorials_utils.py]: I see that you added it later again, maybe squash the arguments so that it is not visible in the history.

File: kso_utils/tutorials_utils.py (hunk 2 above)
Comment (Diewertje11) [reply kso_utils/tutorials_utils.py]: I see that you added it later again, maybe squash the arguments so that it is not visible in the history.

**Review by Diewertje11 — COMMENTED** [review]
I have only 1 question about why something in the test changed. The rest looks good. Only maybe you can squash the commits? Because now I was quite confused why you removed certain things, which I later saw that you added them again.

File: test/widget-tests.py (hunk 1 above)
Comment (jannesgg) [reply test/widget-tests.py:51]: It seems that "Nothing here" was added later. Before it was also "Nothing here" but then later it wasn't an option anymore. So I think there was some change in the underlying csv.

File: kso_utils/tutorials_utils.py (hunk 3 above)
Comment (jannesgg) [reply kso_utils/tutorials_utils.py]: If you go to the "Files changed" tab on the right you will see only the overall changes as well.

File: test/widget-tests.py (hunk 1 above)
Comment (Diewertje11) [reply test/widget-tests.py:51]: Okej :)
As long that we are sure that this is the 'correct' return now for the test.

**Review by Diewertje11 — APPROVED** [review]

File: kso_utils/tutorials_utils.py (hunk 3 above)
Comment (Diewertje11) [reply kso_utils/tutorials_utils.py]: Yes, I know. But when PRs are large it is hard to follow all the changes there. It is easier to follow if every commit has a short explanation with the reasoning. Then one can also see why it is changed and this helps a lot for looking back in the history later as well.
